modules/storage-create-storage-class.adoc

```diff
@@ -19,7 +19,7 @@ usages. By defining a storage class, users can obtain dynamically provisioned
 persistent volumes.
 
 ifeval::["{Provisioner}" == "efs.csi.aws.com"]
-The _AWS EFS CSI Driver Operator_, after being installed, does not create a storage class by default. However, you can manually create the AWS EFS storage class.
+The _Amazon EFS CSI Driver Operator_, after being installed, does not create a storage class by default. However, you can manually create the Amazon EFS storage class.
```

Comment:
Revoked for, Amazon EFS CSI Driver Operator -> AWS EFS CSI Driver Operator.

Reply:
@smunje1 It doesn't seem like this title is linking out to OCP resources or code that reference "AWS EFS". "Amazon EFS CSI driver" is the appropriate name to be used for the driver. Is there a reason I'm missing why this update should be revoked?

https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html

Reply:
Updated, "The _link:https://github.com/openshift/aws-efs-csi-driver-operator[AWS EFS CSI Driver Operator] (a Red Hat operator)_, after being installed, does not create a storage class by default. However, you can manually create the AWS EFS storage class."